tests/unit/multiclass/KNN_unittest.cc

```diff
@@ -226,3 +226,89 @@ TEST(KNN, classify_multiple_kdtree)
 
 
 }
+
+
+TEST(KNN, nearest_neighbours_brute)
+{
+	int32_t feats = 2;
+
+	SGVector< float64_t > lab(5);
+	lab[0] = 0.0; lab[1] = 0.0; lab[2] = 1.0; lab[3] = 1.0; lab[4] = 0.0;
+
+	SGMatrix< float64_t > feat(feats, 5);
```

Comment:
minor: we usually format this without spaces

Reply:
Okay.